nemo/core/connectors/save_restore_connector.py

```diff
@@ -183,6 +183,12 @@ def modify_state_dict(self, conf, state_dict):
                 new_key = key.replace('model.', 'model.module.', 1)
                 new_state_dict[new_key] = state_dict[key]
             state_dict = new_state_dict
+            
```

Comment:
Please do not commit this file as this will affect how other people use this `core` module. Instead please write a custom state_dict function that modifies what gets saved to the .ckpt/.nemo file similar to https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/nlp/models/language_modeling/megatron_gpt_prompt_learning_model.py#L299 if the 'zero shot case' is important for you

Reply:
Please do not commit this file still but no need to write a custom state_dict function